resources/[soz]/soz-core/src/nui/components/Job/OnDutyMenu.tsx

```diff
@@ -0,0 +1,25 @@
+import { FunctionComponent } from 'react';
+
+import { MenuType } from '../../../shared/nui/menu';
+import { MainMenu, Menu, MenuContent, MenuItemText, MenuTitle } from '../Styleguide/Menu';
+
+export const JobOnDutyMenu: FunctionComponent<any> = ({ data }) => {
+    const banner = `https://nui-img/soz/menu_job_${data.job}`;
+
+    if (!data.state.length) {
+        return null;
+    }
+
+    return (
+        <Menu type={MenuType.JobOnDutyMenu}>
+            <MainMenu>
+                <MenuTitle banner={banner}>Employé(e)s en service</MenuTitle>
+                <MenuContent>
+                    {Object.values(data.state).map(player_name => (
+                        <MenuItemText>{`${player_name}`}</MenuItemText>
```

Comment:
Please use this syntax instead of ```{`${player_name}`}```

```suggestion
                        <MenuItemText>{player_name}</MenuItemText>
```

Reply:
Added

Reply:
This solution, even if working, throw an error with eslint
![image](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/46448950/67bf963f-cea8-43a7-acf2-32642d1d552c)

Should I revert it back?

Reply:
No, I fixed this issue. It's related to your type declaration